tests/test_monthly_total_precipitation.py

```diff
@@ -0,0 +1,252 @@
+import datetime
```

Comment:
How much overlap is there between this and ``test_monthly_avgs_of_daily_temp_extrema.py``? Kind of seems *very* similar. Would there be a good way to combine these sets of tests (e.g. appending ``MonthlyTotalPrecipitation`` to the ``views`` list) and testing them together?